src/crawlers/browser_crawler.js

```diff
@@ -251,7 +251,7 @@ class BrowserCrawler extends BasicCrawler {
             handlePageFunction,
             handlePageTimeoutSecs = 60,
             gotoFunction,
-            persistCookiesPerSession,
+            persistCookiesPerSession = true,
```

Comment:
I think @petrpatek decided not to use the default here, because of the check on line 264. Without the default, the user can turn off session pool like that

```js
{
  useSessionPool: false,
}
```

with the default you need

```js
{
  useSessionPool: false,
  persistCookiesPerSession: false,
}
```

I think that's useful. You can fix it by making the check against the original options (I think).

```js
if (!useSessionPool && options.persistCookiesPerSession) {
```

Please also add comment to the check to explain why that is.

Reply:
Yeah, missed this part. I'll do it a bit differently - will remove the default here, and check `persistCookiesPerSession` against specifically `undefined` in `if (useSessionPool) {}`. If it's `undefined` - it's `true`, if not - it would be assigned with the value from options.

+ Would add `else` clause which would set `this.persistCookiesPerSession` to `false` (not to have it `undefined` as you mentioned in the next comment).